src/components/Modal.tsx

```diff
@@ -144,9 +144,26 @@ const CloseButton = styled(Button)`
 `
 
 const ModalContent = styled.div`
-  overflow-y: auto;
-  overflow-x: hidden;
+  /* overflow-y: auto;
+  overflow-x: hidden; */
```

Comment:
This was commented out so that the color picker popup can be properly displayed. Not sure if it has any other implications and wanted to ask if it's ok to remove it before I do.